library/std/src/keyword_docs.rs

```diff
@@ -128,27 +130,27 @@ mod break_keyword {}
 /// const WORDS: &'static str = "hello rust!";
 /// ```
 ///
-/// Thanks to static lifetime elision, you usually don't have to explicitly use 'static:
+/// Thanks to static lifetime elision, you usually don't have to explicitly use `'static`:
 ///
 /// ```rust
 /// const WORDS: &str = "hello convenience!";
 /// ```
 ///
 /// `const` items looks remarkably similar to `static` items, which introduces some confusion as
 /// to which one should be used at which times. To put it simply, constants are inlined wherever
-/// they're used, making using them identical to simply replacing the name of the const with its
-/// value. Static variables on the other hand point to a single location in memory, which all
+/// they're used, making using them identical to simply replacing the name of the `const` with its
+/// value. Static variables, on the other hand, point to a single location in memory, which all
 /// accesses share. This means that, unlike with constants, they can't have destructors, and act as
 /// a single value across the entire codebase.
 ///
-/// Constants, as with statics, should always be in SCREAMING_SNAKE_CASE.
+/// Constants, as with statics, should always be in `SCREAMING_SNAKE_CASE`.
 ///
 /// The `const` keyword is also used in raw pointers in combination with `mut`, as seen in `*const
-/// T` and `*mut T`. More about that can be read at the [pointer] primitive part of the Rust docs.
+/// T` and `*mut T`. More about that usage can be read at the Rust docs for the [pointer primitive].
```

Comment:
Maybe something like this?
```suggestion
/// T` and `*mut T`. The documentation for [pointer primitives] has more information about raw pointers.
```

Reply:
What about:

```suggestion
/// T` and `*mut T`. More about the usage of `const` in raw pointers can be read at the Rust docs for the [pointer primitive].
```

Reply:
Better than mine, but 'the usage of' feels very heavy-handed to me. Maybe just remove it altogether?
```suggestion
/// T` and `*mut T`. More about `const` in raw pointers can be read at the Rust docs for the [pointer primitive].
```

Reply:
I feel like it's a bit clearer though. Maybe this?

```suggestion
/// T` and `*mut T`. More about `const` as used in raw pointers can be read at the Rust docs for the [pointer primitive].
```

But it's not a big deal either way.

Reply:
I like that last one, let's do that :) I've done enough nitpicking :laughing: